frontends/web/src/locales/en/app.json

```diff
@@ -1163,7 +1164,7 @@
     "label": "Your address",
     "onlyThisCoin": {
       "description": "To receive other tokens, enable them in the settings. If you deposit other tokens, they might not be accessible.",
-      "warning": "Make sure to only receive {{accountName}} on this address."
+      "warning": "Make sure to only receive {{coinName}} ({{coinUnit}}) on this address."
```

Comment:
TODO: change in locize

Reply:
imho you can drop coinUnit to spare translation cycles.

did you already change it in locize? make sure to do it in every language. let's do that first before merging.

Reply:
I'll do it now.